silverstripe/framework/CVE-2023-48714.yaml

```diff
@@ -0,0 +1,11 @@
+title:     "CVE-2023-48714 Record titles for restricted records can be viewed if exposed by GridFieldAddExistingAutocompleter"
+link:      https://www.silverstripe.org/download/security-releases/CVE-2023-48714
+cve:       CVE-2023-48714
+branches:
+    4.13.x:
```

Comment:
according to the linked report all 3.x versions are affected too

Reply:
Ahh yup good catch, thanks. I keep forgetting that exists since it's EOL.
Updated.